drivers/gpio/gpio_ite_it8xxx2.c

```diff
@@ -481,10 +481,12 @@ static int gpio_ite_get_config(const struct device *dev,
 	reg_1p8v = &IT8XXX2_GPIO_GCRX(
 			gpio_1p8v[gpio_config->index][pin].offset);
 	mask_1p8v = gpio_1p8v[gpio_config->index][pin].mask_1p8v;
-	if (*reg_1p8v & mask_1p8v) {
-		flags |= IT8XXX2_GPIO_VOLTAGE_1P8;
-	} else {
-		flags |= IT8XXX2_GPIO_VOLTAGE_3P3;
+	if (reg_1p8v != &IT8XXX2_GPIO_GCRX(0)) {
```

Comment:
To the casual reader, it does not feel intuitively obvious why this condition is being made.  I would personally prefer to see a short comment so the meaning is clear.

Reply:
Done